test/micro/org/openjdk/bench/java/lang/foreign/libToCString.c

```diff
@@ -21,13 +21,14 @@
  * questions.
  */
 
+#include <jlong_md.h>
```

Comment:
I think you just want `jlong.h` here? AFAIK the `jlong_md.h` is the platform specific implementation file, whereas `jlong.h` is the 'interface'.

Reply:
Right. I copy-pasted from some other file that includes `jlong_md.h` instead of `jlong.h`. For cleanliness reasons, we should of course include `jlong.h`. Fixed in new commit.